ui/src/app/components/environment-informed-action-panel.tsx

```diff
@@ -76,7 +76,9 @@ export const EnvironmentInformedActionPanel = ({
   environmentChanged = false,
 }) => (
   <FlexRow style={styles.environmentInformedActionPanelWrapper}>
-    <StartStopEnvironmentButton {...{ status, onPause, onResume, appType }} />
+    {appType === UIAppType.JUPYTER && (
+      <StartStopEnvironmentButton {...{ status, onPause, onResume, appType }} />
+    )}
```

Comment:
We don't have a "pause/resume Cromwell from config panel" test so there are no unit tests to change here.